### ISSUE TITLE
Issue with the dependency `org.reflections:reflections:jar:0.9.12`

### ISSUE THREAD

**Issue by gianvitom** [zalando-stups/java-sproc-wrapper#98]
It looks there is an issue due to the dependency `org.reflections:reflections:jar:0.9.12` when the library is used in a Dockerised environment.

The error (on Spring Boot startup) is:
```
java.lang.IllegalStateException: zip file closed
at java.util.zip.ZipFile.ensureOpen(Unknown Source) ~[?:?]
at java.util.zip.ZipFile.jarStream(Unknown Source) ~[?:?]
...
at org.reflections.vfs.ZipDir.lambda$getFiles$2(ZipDir.java:21) ~[reflections-0.9.12.jar!/:?]
```

Testing the previous version of the `java-sproc-wrapper` with version 3.0.1, it is working.

But from the 3.1.0, it gives that error.

**Comment by vadeg** [comment]
Hi @gianvitom,

Could you provide more details about the stacktrace? From where the `org.reflections.vfs.ZipDir.lambda$getFiles` is called from in the library?

**Comment by vadeg** [comment]
The issue may be related to ronmamo/reflections#279

**Comment by vadeg** [comment]
Fixed in #104
